Build issue for v0.8.1

Getting these errors when trying to build. I had previously been on v0.8.0 with no build issues. My ESP-IDF version is `v5.4-dev-3201-g46acfdce96`.
[build-output.txt](https://github.com/user-attachments/files/17452919/build-output.txt)
[Cargo.txt](https://github.com/user-attachments/files/17452923/Cargo.txt)
[sdkconfig.txt](https://github.com/user-attachments/files/17452926/sdkconfig.txt)

I am trying to build with IDF version v5.2.2 and am facing this error : 

`error[E0432]: unresolved imports `esp_idf_sys::ble_uuid_any_t`, `esp_idf_sys::ble_uuid_cmp`
 --> /home/srijan/.cargo/git/checkouts/esp32-nimble-fc9d2c54a3bfa7fe/980873d/src/server/ble_characteristic.rs:5:19
  |
5 | use esp_idf_sys::{ble_uuid_any_t, ble_uuid_cmp};
  |                   ^^^^^^^^^^^^^^  ^^^^^^^^^^^^ no `ble_uuid_cmp` in `sys`
  |                   |
  |                   no `ble_uuid_any_t` in `sys`
`

am I missing something?

--- a/build.rs
+++ b/build.rs
@@ -11,4 +11,5 @@ fn main() {
   println!("cargo::rustc-check-cfg=cfg(esp_idf_version_minor, values(\"2\"))");
   println!("cargo::rustc-check-cfg=cfg(esp_idf_version_patch, values(\"0\"))");
   println!("cargo::rustc-check-cfg=cfg(esp_idf_version_patch, values(\"1\"))");
+  println!("cargo::rustc-check-cfg=cfg(esp_idf_version_patch, values(\"2\"))");
 }

--- a/src/server/ble_characteristic.rs
+++ b/src/server/ble_characteristic.rs
@@ -23,7 +23,7 @@ cfg_if::cfg_if! {
   if #[cfg(all(
     esp_idf_version_major = "5",
     esp_idf_version_minor = "2",
-    not(any(esp_idf_version_patch = "0", esp_idf_version_patch = "1"))
+    not(any(esp_idf_version_patch = "0", esp_idf_version_patch = "1", esp_idf_version_patch="2"))
   ))] {
     type NotifyTxType = esp_idf_sys::ble_gap_event__bindgen_ty_1__bindgen_ty_12;
     type Subscribe = esp_idf_sys::ble_gap_event__bindgen_ty_1__bindgen_ty_13;

